rails-type-casting.md

```diff
@@ -0,0 +1,62 @@
+Title
+==
+
+I don't know I'm bad at naming things
```

Comment:
Maybe something along the lines of "You don't need magic to write Ruby" or something like that.